engines/glk/detection.cpp

```diff
@@ -230,4 +230,9 @@ void GlkMetaEngineDetection::detectClashes() const {
 #endif
 }
 
+uint GlkMetaEngineDetection::getMD5Bytes() const {
+	// TODO: Implement this properly
```

Comment:
This requires refactoring since currently the engine hardcodes it:

```C
adrift/detection.cpp:           Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
advsys/detection.cpp:           Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
agt/detection.cpp:              Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
alan2/detection.cpp:            Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
alan3/detection.cpp:            Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
archetype/detection.cpp:                Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
comprehend/charset.cpp: Common::String md5 = Common::computeStreamMD5AsString(f, 1024);
comprehend/detection.cpp:               Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
comprehend/game_oo.cpp: Common::String md5 = Common::computeStreamMD5AsString(f, 1024);
comprehend/game_tm.cpp: Common::String md5 = Common::computeStreamMD5AsString(f, 1024);
glulx/detection.cpp:            Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
hugo/detection.cpp:             Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
jacl/detection.cpp:             Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
magnetic/detection.cpp:         Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
metaengine.cpp:         gameDesc._md5 = Common::computeStreamMD5AsString(f);
metaengine.cpp:         gameDesc._md5 = Common::computeStreamMD5AsString(f, 5000);
quest/detection.cpp:            Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
scott/detection.cpp:                    md5 = Common::computeStreamMD5AsString(gameFile);
scott/detection.cpp:                    md5 = Common::computeStreamMD5AsString(gameFile, 5000);
tads/detection.cpp:             Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
zcode/detection.cpp:            Common::String md5 = Common::computeStreamMD5AsString(gameFile, 5000);
```

Do you plan to work on it?

Reply:
Though, after a closer look, I think, it could be just hardcoded to 5000 bytes :D

Reply:
Pushed the aforementioned change